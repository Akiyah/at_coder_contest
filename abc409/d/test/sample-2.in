8
7
atcoder
1
x
5
snuke
5
aaaaa
5
aaaab
5
aaaba
5
abcde
5
abcea
