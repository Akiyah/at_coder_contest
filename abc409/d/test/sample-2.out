acodert
x
nsuke
aaaaa
aaaab
aaaab
abcde
abcae
